spec/lib/find_specific_term_spec.rb

```diff
@@ -24,5 +26,19 @@
 
       FindSpecificTerm.call(term)
     end
+
+    it "does not find content item if document type is excluded" do
```

Comment:
Minor: could we also test that an eligible document will return when excluding certain doc types?